protected/humhub/modules/space/widgets/views/membershipButton.php

```diff
@@ -4,27 +4,39 @@
 use humhub\modules\space\models\Membership;
 use yii\helpers\Html;
 
+/* @var $membership Membership */
+/* @var $space Space */
+/* @var $options array */
+/* @var $canCancelMembership bool */
+
 if ($membership === null) {
     if ($space->canJoin()) {
         if ($space->join_policy == Space::JOIN_POLICY_APPLICATION) {
-            echo Html::a(Yii::t('SpaceModule.base', 'Request membership'), $space->createUrl('/space/membership/request-membership-form'), ['id' => 'requestMembershipButton', 'class' => 'btn btn-primary', 'data-target' => '#globalModal']);
+            echo Html::a($options['requestMembership']['title'], $options['requestMembership']['url'], $options['requestMembership']['attrs']);
         } else {
-            echo Html::a(Yii::t('SpaceModule.base', 'Become member'), $space->createUrl('/space/membership/request-membership'), ['id' => 'requestMembershipButton', 'class' => 'btn btn-primary', 'data-method' => 'POST']);
+            echo Html::a($options['becomeMember']['title'], '#', $options['becomeMember']['attrs']);
         }
     }
 } elseif ($membership->status == Membership::STATUS_INVITED) {
     ?>
-    <div class="btn-group">
-        <?= Html::a(Yii::t('SpaceModule.base', 'Accept Invite'), $space->createUrl('/space/membership/invite-accept'), ['class' => 'btn btn-info', 'data-method' => 'POST']); ?>
-        <button type="button" class="btn btn-info dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+    <div class="<?= $options['acceptInvite']['groupClass'] ?>">
+        <?= Html::a($options['acceptInvite']['title'], '#', $options['acceptInvite']['attrs']); ?>
+        <button type="button" class="<?= $options['acceptInvite']['togglerClass'] ?> dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
             <span class="caret"></span>
             <span class="sr-only">Toggle Dropdown</span>
```

Comment:
Please add translation

Reply:
@luke- Do you think we should translate text for screen readers? https://getbootstrap.com/docs/4.0/utilities/screenreaders/

If yes, then should we translate this in all other places? :

<img width="916" alt="sr-only" src="https://user-images.githubusercontent.com/6995524/120808494-cc90da00-c551-11eb-85b0-078fad652b34.png">

Reply:
@yurabakhtin Oh ok, then let this as it is for now!